css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,41 @@
+.container {
+  flex-wrap: wrap;
```

Comment:
`flex-wrap` property is only used on a flex parent set to `display: flex`. This property can be removed.